cmd/ratify/cmd/referrer.go

```diff
@@ -181,6 +181,10 @@ func showRefManifest(opts referrerCmdOptions) error {
 		return err
 	}
 
+	if subRef.Digest == "" {
+		fmt.Println(taggedReferenceWarning)
```

Comment:
I think we should return error in the future. For now I think it makes sense to follow notation CLI which just warns for now.

Reply:
I didn't add it to the util function since it seems like the util function is used in many different places in the executor and there might be duplicate messages that appear. Also for CLI the output would be slightly different (printf vs logging). And the resolve cli command also uses the same util and it probably wouldn't make sense to warn for this.